lib/Analysis/Allocation.cpp

```diff
@@ -51,13 +51,29 @@ getCvtOrder(const Attribute &srcLayout, const Attribute &dstLayout) {
   return {inOrd, outOrd};
 }
 
+bool isMmaToDotShortcut(MmaEncodingAttr &mmaLayout,
```

Comment:
There exist multiple copies of `isMmaToDotShortcut` (See `ConvertLayoutOpToLLVM.cpp` and `TritonGPUToLLVMBase.h`, we may want to merge them together into `Utility.cpp`?

Reply:
Good catch. Just cleaned that up.